dll/win32/shlwapi/string.c

```diff
@@ -2878,6 +2878,9 @@ HRESULT WINAPI SHLoadIndirectString(LPCWSTR src, LPWSTR dst, UINT dst_len, void
     WCHAR *dllname = NULL;
     HMODULE hmod = NULL;
     HRESULT hr = E_FAIL;
+#ifdef __REACTOS__
+    WCHAR szExpanded[MAX_PATH];
```

Comment:
I am not sure if `MAX_PATH` will be long enough here.

Reply:
`512` is enough.